pkg/discovery/interface.go

```diff
@@ -88,3 +88,20 @@ func CreateGroupDiscovery(pd configtypes.PluginDiscovery, criteria *GroupDiscove
 	}
 	return nil, errors.New("unknown group discovery source")
 }
+
+// PluginDiscoveryOpts used to customize the plugin discovery process or mechanism
+type PluginDiscoveryOpts struct {
+	ForceCache bool // ForceCache used to pull the plugin data from the cache
```

Comment:
I feel the `ForceCache` variable name (along with the specified comment) does not justify the meaning of this variable. One could interpret the variable as `ForceCacheUpdate` or `UseCacheOnly`.

I feel we want to say UseCacheOnly, so renaming the variable to something like `SkipCacheUpdate` or `UseLocalCacheOnly` would give more meaning to the variable name.